source/includes/connect/tls-ca-file.c

```diff
@@ -0,0 +1,21 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main (void) {
+
+  // start-connect
+  // Initialize the MongoDB C Driver
+  mongoc_init ();
+  
+  // Create a new client, set TLS option to true, set path to client certificate file
```

Comment:
S: Since you're already describing the code in the rST, I'd remove this comment or move it outside the //start- and //end- comments

Reply:
Applies to all the code files